megamek/src/megamek/client/ui/swing/CustomMechDialog.java

```diff
@@ -496,7 +510,28 @@ public CustomMechDialog(ClientGUI clientgui, Client client, List<Entity> entitie
 
             fldStartAltitude.setText(new Integer(entity.getAltitude()).toString());
             fldStartAltitude.addActionListener(this);
+            
+            fuel = a.getFuel();
+            fldCurrentFuel.setText(new Integer(a.getCurrentFuel())
+                    .toString());
+            fldCurrentFuel.addActionListener(this);
         }
+        
+        if (isShip) {
```

Comment:
Is this block of code necessary?  Isn't the `isAero || isLam` block above exactly the same?  Couldn't you juse have `isAero || isLam || isShip`?  It's best to avoid code duplication.

Reply:
It probably can all be together. I think this code is a holdover from the first time through when I was having issues making my code work with IAero. I'll check.

Reply:
Fixed. It did work ok all as one block.